rc/tools/make.kak

```diff
@@ -53,13 +61,13 @@ define-command -hidden make-open-error -params 4 %{
 define-command -hidden make-jump %{
     evaluate-commands %{
         try %{
-            execute-keys gl<a-?> "Entering directory" <ret><a-:>
+            execute-keys gl<a-?> %opt{make_dir_pattern} <ret><a-:>
```

Comment:
Putting this in an execute key is dangerous as and `<foo>` in the regex might get misinterpretted as a key spec. The correct way is to `set-register slash %opt{make_dir_pattern}` then `execute-keys gl<a-?><ret><a-:>`.

Reply:
Will do